README.md

```diff
@@ -36,150 +49,158 @@ terraform init -upgrade
 to upgrade to the latest stable version of the vra7 provider. See the [Terraform website](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
 for more information on provider upgrades, and how to set version constraints on your provider.
 
-## Configure
-The VMware vRA terraform configuration file contains two objects.
+Migrating from previous versions to version 1.0.0, issues fixed and new features
+---------------------------------------------------------------------------------
 
-### Provider
-This part contains service provider details.
+There are some schema changes in the provider version 1.0.0. These changes are made to support vra7 deployment Day 2 actions, detailed information of the deployment in the state file, getting access to more deployment and resource level properties, esp. `ip_address`, etc. See the release notes for more detail.
 
-Provider block contains four mandatory fields:
-* `username` - vRA portal username
-* `password` - vRA portal password
-* `tenant` - vRA portal tenant
-* `host` - End point of REST API
-* `insecure` - In case of self-signed certificates, default value is false
+# Previous main.tf file
 
-**Example:**
+hcl
```

Comment:
Throughout this file the hcl syntax hint needs to be at the end of the starting triple backticks.